src/components/accueil/Accueil.tsx

```diff
@@ -59,6 +66,55 @@ export const Accueil = () => {
     return activitiesWithLastMimic;
   }
 
+  const getType = (typeValue: number): string | undefined => {
+    const type = Object.keys(ActivityType).find((key) => ActivityType[key] === typeValue);
+    return type;
+  };
+
+  const SUBTYPE_MAPPER = {
+    ENIGME: ENIGME,
+    INDICE: INDICE,
+    SYMBOL: SYMBOL,
+    DEFI: DEFI,
+    REPORTAGE: REPORTAGE,
+  };
+  const getSubtype = (typeName: string, subTypeValue: number): string | undefined => {
+    const result = Object.keys(SUBTYPE_MAPPER[typeName] || {}).find((key) => SUBTYPE_MAPPER[typeName][key] === subTypeValue);
+    return result;
+  };
+
+  function filterActivitiesByTerm(activitiesData: Activity<AnyData>[]): Activity<AnyData>[] {
```

Comment:
**Nit:** I would prefer the name `activities` because you take a list of activities

Reply:
the name `activities `already exists, `filterActivities` is ok fo you ?